packages/amplify-e2e-tests/src/__tests__/resolvers.test.ts

```diff
@@ -104,6 +105,66 @@ describe('user created resolvers', () => {
         ]
       `);
     });
+
+    it('disable resolver deduping using transform.config.json file', async () => {
+      await addApiWithoutSchema(projectDir, { apiName });
+      updateApiSchema(projectDir, apiName, 'two_models_with_cognito_auth.graphql');
+      await amplifyPushGraphQlWithCognitoPrompt(projectDir);
+      await apiGqlCompile(projectDir);
+      
+      // With default behavior, functions in Author stack will reference to functions to Todo stack
+      const authorJsonPath = join(projectDir, 'amplify', 'backend', 'api', apiName, 'build', 'stacks', 'Author.json');
+      const authorJsonBefore = JSON.parse(fs.readFileSync(authorJsonPath).toString());
+      expect(authorJsonBefore.Resources.GetAuthorResolver.Properties.PipelineConfig.Functions).toMatchInlineSnapshot(`
+        Array [
+          Object {
+            "Ref": "referencetotransformerrootstackTodoNestedStackTodoNestedStackResource9AC126A3OutputstransformerrootstackTodoQuerygetTodoauth0FunctionQuerygetTodoauth0FunctionAppSyncFunction9F8E8363FunctionId",
```

Comment:
Are the hashed portions of these keys stable across compilations?

Reply:
This could probably simplified (and improved from a readability perspective to be something like:

```ts
expect(authorJsonBefore.Resources.GetAuthorResolver.Properties.PipelineConfig.Functions).toEqual(expect.arrayContaining([
  { Ref: expect.stringContaining('getTodoauth0Function') },
  { Ref: expect.stringContaining('getTodopostAuth0Function') },
  { Ref: expect.stringContaining('GetAuthorDataResolverFn') },
]));
```